speedtest/speedtest.yml

```diff
@@ -1,35 +1,30 @@
 apiVersion: influxdata.com/v2alpha1
 kind: Label
 metadata:
-    name: adoring-hellman-62a001
+    name: crumbling-lovelace-a76001
 spec:
     color: '#0f0e15'
     name: speedtest
 ---
 apiVersion: influxdata.com/v2alpha1
 kind: Bucket
 metadata:
-    name: blissful-antonelli-62a003
+    name: serene-mendel-e76003
 spec:
     associations:
       - kind: Label
-        name: adoring-hellman-62a001
+        name: crumbling-lovelace-a76001
     name: speedtest
 ---
 apiVersion: influxdata.com/v2alpha1
 kind: Dashboard
 metadata:
-    name: condescending-davinci-a2a001
+    name: wonderful-aryabhata-676001
 spec:
     associations:
       - kind: Label
-        name: adoring-hellman-62a001
+        name: crumbling-lovelace-a76001
```

Comment:
do these names need to change? this would block anyone from upgrading this template if they have it already installed.

Reply:
Only for curious, why is that behavior? The template that is hosted today isn't working because the speedtest-cli changed, so, the idea is to renew the dashboard and make it compatible with the new version of Speedtest-cli.

Reply:
i should have commented on line 22 instead. if the current implementation is broken and no one is using it, then this isn't really an issue. 

consider the use case where a user has a stack for a specific template. if they make an update to the template, and want to apply that update to the same stack, the system uses the names in the template to understand if it needs to update the dashboard or create a new one.

you can't just use the actual name since it's not unique in the org.

Reply:
Ok. Is it still needed to maintain the name of the labels, or can it be merged?

Reply:
nah, let's merge it!